docs/manual/getting_started.rst

```diff
@@ -581,51 +686,51 @@ Data-Centric Architecture
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
 In a service-centric architecture, to interact, applications need to
-know each other's interfaces to share data, share events, share
-commands, or replies. These interfaces are modeled as sets of operations
+know each other's interfaces to share data, share events, and share
+commands or replies. These interfaces are modeled as sets of operations
 and functions that are managed in centralized repositories. This kind of
-architecture creates unnecessary dependencies that end-up creating a
+architecture creates unnecessary dependencies that create a
 tightly coupled system. The centralized interface repositories are
 usually seen as a single point of failure.
 
 In a data-centric architecture, your design focuses on the data each
-application produces and decides to share rather than on the Interfaces'
+the application produces and decides to share rather than on the Interfaces'
 operations and the internal processing that produced them.
 
 A data-centric architecture creates a decoupled system that focuses on
 the data and applications states' that need to be shared rather than the
 applications' details. In a data-centric system, data and their
-associated quality of services are the only contract that bounds the
+associated quality of services are the only contracts that bounds the
 applications together. With DDS, the system decoupling is
-bi-dimensional, in Space and in Time.
+bi-dimensional, in Space and time.
 
 Space-decoupling derives from the fact that applications do not need to,
```

Comment:
GitHub is preventing me from suggesting code now for some reason due to "deleted lines". So I will write my suggestion here:
Space-decoupling derives from the fact that applications do not need to know the identity of the data produced or consumed, nor their logical or physical location in the network.